### DISCUSSION TITLE
bullet3 release

### DISCUSSION THREAD

**Discussion by ahundt** [bulletphysics/bullet3#3559] [General]
@ilovezfs had pinged me because I've got a pending homebrew installer update https://github.com/Homebrew/homebrew-core/pull/13388, which I wrote a while ago for bullet3 (as opposed to pybullet). It is waiting on a release of bullet proper, is there perhaps a timeline so I can answer his question?

**Comment by erwincoumans** [comment]
Good point, I'll tag a new Bullet release soon, together with a new
pybullet release that contains all the new gym environments build-in (pip
install pybullet is going to also install the pybullet_envs package).

**Comment by ahundt** [comment]
Sounds great, thanks!
